Training25/Program.cs

```diff
@@ -3,11 +3,31 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program to reduce a string of lowercase characters by deleting a pair of adjacent letters that
+// match.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
+
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      while (true) {
+         Write ("Reduced String\n~~~~~~~~~~~~~~\nEnter a lowercase string: ");
+         var input = ReadLine ();
+         if (input is null) {

```

Comment:
```suggestion
         if (string.IsNullOrEmpty (input)) { WriteLine ("Invalid input!"); continue; }
```
Why only null is checked? Also, keep such simple body in a single line.